pandas/tests/io/formats/test_format.py

```diff
@@ -3377,14 +3376,16 @@ def test_filepath_or_buffer_arg(
     filepath_or_buffer_id,
 ):
     df = DataFrame([data])
+    if method in ["to_latex"]:  # uses styler implementation
```

Comment:
Does it make sense to simplify it to ``method == 'to_latex'``?

Reply:
if other methods move over to Styler, (the intention being `to_html`) we will need a list here anyway